### PR TITLE
Adding wiring-pi definition

### PR DESCRIPTION
**Pull request by NoHomey**
case 1. Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

Adding a new type definition for [`wiring-pi`](https://github.com/eugeneware/wiring-pi)

API Reference: https://github.com/eugeneware/wiring-pi/blob/master/DOCUMENTATION.md#apis

### CODE REVIEW

File: wiring-pi/wiring-pi.d.ts [hunk 1]
```diff
@@ -0,0 +1,314 @@
+// Type definitions for Jest 2.1.1
```
Comment (andy-ms) [inline wiring-pi/wiring-pi.d.ts]: Wrong project name

**Review by ghost — COMMENTED** [review]
Looks good. Could you clean up the git history by rebasing?

**Comment by vvakame** [comment]
@andy-ms I'm using `Squash and merge` button in this repo. https://help.github.com/articles/about-pull-request-merges/
default settings are separated by repository. It is very useful.

**Comment by andy-ms** [comment]
@vvakame Thanks for the help! Still waiting on project name fix though.

**Comment by NoHomey** [comment]
@andy-ms I've done all you've requested.

**Review by ghost — APPROVED** [review]

**Pull request merged**